packages/orbit-components/src/Tag/index.js

```diff
@@ -19,103 +20,106 @@ const getFontSize = ({ theme, size }) => {
-  transition: color ${({ theme }) => theme.orbit.durationFast} ease-in-out,
-    box-shadow ${({ theme }) => theme.orbit.durationFast} ease-in-out,
-    background ${({ theme }) => theme.orbit.durationFast} ease-in-out;
-
-  &:focus {
-    outline: 0;
-  }
-
-  ${({ actionable }) =>
-    actionable &&
-    css`
-      cursor: pointer;
-
-      &:hover {
-        background: ${getBackgroundColor(STATES.HOVER)};
-        box-shadow: none;
-      }
-
-      &:active {
-        background: ${getBackgroundColor(STATES.ACTIVE)};
-        box-shadow: none;
-      }
-
-      &:focus {
-        background: ${getBackgroundColor(STATES.HOVER)};
-        box-shadow: none;
-        outline: 0;
-      }
-    `};
-`;
-
-StyledTag.defaultProps = {
-  theme: defaultTheme,
-};
-
-const IconContainer = styled.div`
+const CloseContainer = styled.div`
   display: flex;
   margin-${left}: 8px;
+  opacity: 0.5;
+  color: ${resolveColor({
+    removable: "paletteWhite",
+    selected: "paletteBlueDarker",
+    normal: "paletteInkLink",
+  })};
+  cursor: ${({ actionable }) => actionable && `pointer`};
+  transition: color ${({ theme }) => theme.orbit.durationFast} ease-in-out;
 
-  svg {
-    height: ${({ theme }) => theme.orbit.widthIconSmall};
-    width: ${({ theme }) => theme.orbit.heightIconSmall};
+  &:active {
+    color: ${({ theme, selected }) =>
+      selected ? theme.orbit.paletteBlueDarker : theme.orbit.paletteWhite};
   }
 `;
 
-IconContainer.defaultProps = {
+CloseContainer.defaultProps = {
   theme: defaultTheme,
 };
 
-const CloseContainer = styled.div`
-  display: flex;
-  margin-${right}: 8px;
-  color: ${({ theme, selected }) =>
-    selected ? theme.orbit.paletteBlueDarker : theme.orbit.paletteInkLight};
-  cursor: pointer;
-  transition: color ${({ theme }) => theme.orbit.durationFast} ease-in-out;
-
-  &:hover {
-    color: ${({ theme, selected }) =>
-      selected ? theme.orbit.paletteBlueDarker : theme.orbit.paletteInkLight};
-  }
-  &:active {
-    color: ${({ theme, selected }) =>
-      selected ? theme.orbit.paletteBlueDarker : theme.orbit.paletteInkLight};
-  }
+export const StyledTag = styled.div`
+  ${({ theme }) => css`
+    font-family: ${theme.orbit.fontFamily};
+    color: ${resolveColor({
+      removable: "paletteWhite",
+      selected: "paletteBlueDarker",
+      normal: "colorTextTag",
+    })};
+    background: ${getBackgroundColor(STATES.DEFAULT)};
+    display: inline-flex;
+
+    box-sizing: border-box;
+    justify-content: center;
+    align-items: center;
+    font-size: ${getFontSize};
+    font-weight: ${theme.orbit.fontWeightMedium};
+    border-radius: ${theme.orbit.borderRadiusNormal};
+    padding: ${rtlSpacing(theme.orbit.paddingTag)};
+    transition: color ${theme.orbit.durationFast} ease-in-out,
+      box-shadow ${theme.orbit.durationFast} ease-in-out,
+      background ${theme.orbit.durationFast} ease-in-out;
+
+    &:focus {
+      outline: 0;
+    }
+
+    ${({ actionable }) =>
```

Comment:
Could be moved to the first function in the top - if we want to go with one function 🙂